### DISCUSSION TITLE
1.28.1

### DISCUSSION THREAD

**Discussion by Koenkk** [Koenkk/zigbee2mqtt#14760] [General]
# Improvements
- [#603](https://github.com/Koenkk/zigbee-herdsman/pull/603) Support Zigbee Green Power for EZSP adapters
- [#4733](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4733) Add `emergency_heating` to `system_mode` of Hive SLR2b/c
- #14313 Discovery Home Assistant numeric sensors with `value_step` value if set
- [#593](https://github.com/Koenkk/zigbee-herdsman/pull/593) Support EZSP adapter path auto detection for Home Assistant SkyConnect and Sonoff dongle Plus-E
- #14358 Mark `min_temperature_limit` as a config variable for Home Assistant
- [#4744](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4744) Support `min_temperature_limit` for Moes BHT-002
- [#4745](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4745) Support `energy` for TuYa TS0001_power
- [#4747](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4747) Expose climate `sensor` as separate entity
- #13438 Support effects for Home Assistant light groups
- #14396 Allow Zigbee2MQTT to be installed as a module
- [#4743](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4734) Support `occupied_heating_setpoint_scheduled` for Danfoss 014G2461
- [#4754](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4754) Support external sensor for Xiaomi SRTS-A01
- [#4756](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4756) Support more actions for Müller Licht 404022/404049C
- [#4757](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4757) Add OTA support for Perenio `PEHPL01`, `PECMS01` and `PECLS01`
- [#4762](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4762) Support `very_high` and `max` `motion_sensitivity` for Philips 9290030675 and 9290030674
- [#4735](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4735) Allow to configure schedule for Moes BHT-002 thermostat
- [#4776](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4776) Support more features for Sunricher ZG9092A and fix time sync
- #12793  Merge `_TZE200_579lguh2` (X706U) and `_TZE200_la2c2uo9` (MS-105Z) with TuYa TS0601_dimmer_1 to support extra functionality
- #14502 Expose step size of `local_temperature_calibration` Home Assistant sensor
- #14471 Expose temperature and fix energy for ROBB ROB_200-017-0
- [#4807](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4803) Expose more features for TuYa TS0601_gas_sensor
- [#4814](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4814) Support indicator_mode for Moes ZP-LZ-FR2U
- [#4815](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4815) Expose separate calibration_time for right endpoint of Lonsonho TS130F_dual
- [#4750](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4750) TuYa TV02-Zigbee rework
- #14625 Set `NODE_ENV` to `PRODUCTION` in Docker container
- [#4831](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4831) Expose additional functionality for Nous SZ-T04
- [#4813](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4813) Remove `ballast_physical_minimum_level`/`ballast_physical_maximum_level` since it is not settable and very specific
- #1172 Expose additional features for Xiaomi ZNCZ12LM
- [#4819](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4819) Support OTA and expose action for OSRAM 4062172044776
- [#4791](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4791) Rely on power measurements reporting instead of polling for Lidl HG08673-FR
- [#4832](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4832) Support `system_mode` and `running_state` for Moes BRT-100-TRV
- [#4859](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4859/) Support `fall_down_status` for TuYa MIR-HE200-TY (`_TZE200_vrfecyku`)
- [#4861](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4861) Support `system_mode` for TuYa TS0601_thermostat_1

# Fixes
- #13708 Fix configure error for TuYa TS0202 `_TYZB01_71kfvvma`
- #13767 Fix support for TuYa DIN energy meter `_TZE200_lsanae15` and `_TZE200_bkkmqmyo`
- [#4741](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4741) Fix configure failing for Develco MOSZB-140
- #14359 Fix battery % multiplied by two for Nyce NCZ-3043-HA
- #8072 Fix fast discharge and non-responsive Tuya TS0041, TS0041, TS0042, TS0043
- [#4760](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4760) Rename `state` to `system_mode` to fix Home Assistant integration
- [#4770](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4770) Fix Legrand 067776  cover state
- #13991 Fix `null` values for LiXee ZLinky_TIC
- #14495 Fix HEIMAN HS3CG gas reporting
- #14523 Fix channel control of TuYa TS0003 and TS0004
- #14470 Fix `INVALID_DATA_TYPE` error when setting Develco ZHEMI101 interface mode
- [#4795](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4795) Fix wrong window status in Home Assistant for Moes BRT-100-TRV
- [#4801](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4801) Fix energy reporting for Inovelli VZM35-SN
- [#4802](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4802) Fix LED effects for the Inovelli VZM31SN and a few other attribute
- #13990 Fix Osram 4052899926110 color not controllable
- #14580 Fix incorrect max color temperature of AwoX 33957 (370 -> 454)
- #14277 Fix status of Develco MOSZB-140 not updating
- #14612 Fix battery % of Perenio PECLS01 multiplied by two
- [#4821](https://github.com/Koenkk/zigbee-herdsman-converters/issues/4821) Fix `trv_mode` not in state after setting it for Eurotronic SPZB0001
- [#4824](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4824) Fix color temp range of HORNBACH 10011723 and Immax 07089L
- [#4833](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4833) Fix Sprut.device WB-MSW-ZIGBEE v.3 configure failing sometimes
- #12400 Fix Immax 07048L not reporting power
- [#4839](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4839) Fix TuYa JM-TRH-ZGB-V1 and Nous E6 and SZ-T04 `temperature_alarm` and `humidity_alarm` value
- #14256 Fix missing actions and constant battery % reading for Legrand 067774 and 067773
- [#4852](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4852) Remove unsupported `occupied_heating_setpoint` and `system_mode` from Plugwise 106-03

# New supported devices
This release adds support for **34** new devices:
- `02973.B` Vimar Vimar IoT thermostat
- `3RTHS24BZ` Third Reality Temperature and humidity sensor
- `5110731H5` Philips Hue white ambiance Garnea downlight
- `540139X` Namron Panel heater 400/600/800/1000 W
- `71150` OSRAM Lightify under cabinet tunable white
- `81810` AduroSmart Zigbee Aduro Eria B22 bulb - warm white
- `8718696174548` Philips  Hue Sana wall lamp with Bluetooth white
- `8718801528273` Ynoa Smart LED GU10 CCT
- `871951433908` Philips Hue white ambiance Buckram double spotlight with Bluetooth
- `8719514339125` Philips Hue white ambiance Buckram triple spotlight with Bluetooth
- `8719514339163` Philips Hue white ambiance Buckram quadruple spotlight with Bluetooth
- `9290018189` Philips Hue white PAR38 outdoor
- `929003054601` Philips Hue white ambiance Fair ceiling with Bluetooth white
- `C0040000` Villeroy & Boch Zigbee home automation kit for mirror
- `C204` Candeo Zigbee micro smart dimmer
- `EFR32MG21.Router` Custom devices (DiY) EFR32MG21 router
- `IM-CDZDGAAG005KA-MZN` ADEO ENKI LEXMAN GU-10 LED white
- `IOMZB-110` Develco IO module
- `PMM-300Z3` ShinaSystem SiHAS 3phase energy monitor
- `RB 279 T` Innr Smart bulb tunable white E27
- `RC 250` Innr Remote control
- `S093TH-ZG` HZC Electric Temperature and humidity sensor
- `SIN-4-RS-20_PRO` NodOn Roller shutter controller
- `SPPUSB02` Mercator Ikuü double power point with USB
- `SR-ZG9001K12-DIM-Z5` Sunricher 5 zone remote and dimmer
- `SZ-PIR04N` Sercomm PIR motion & temperature sensor
- `TS0202_2` TuYa Motion sensor with scene switch
- `TS0601_dimmer_2` TuYa 2 gang smart dimmer
- `TS0601_dimmer_3` TuYa 3 gang smart dimmer
- `TS0601_din_1` TuYa Zigbee DIN energy meter
- `TS0601_smoke_1` TuYa Smoke sensor
- `TS0601_switch_5_gang` TuYa 5 gang switch
- `WF4C_WF6C` Acuity Brands Lighting (ABL) Juno 4" and 6" LED smart wafer downlight
- `WXKG20LM` Xiaomi Aqara E1 wireless mini switch

<hr /><em>This discussion was created from the release <a href='https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.28.1'>1.28.1</a>.</em>

**Comment by datsrt02** [comment]
Sr sir, i saw one of them new devices supported is "02973.B Vimar Vimar IoT thermostat". I have one it, but i see the manual of 02973 in vimar.com website, i cant find where the guide how to configure in zigbee mode, i say working only bluetooth mode, so can you tell me how to configure 02973 in zigbee mod , to pair with zigbee2mqtt addon ?
